mars/cell_lysis.lfr

```diff
@@ -0,0 +1,22 @@
+module cell_lysis (
+    finput A, B, C, D,
+    foutput a_out
+);
+
+    flow mixture1, mixture2;
+
+    // mix suspended cells and lysing buffer
+    assign mixture1 = A + B;
+
+    // mix mixture1 with neutralization buffer to prevent DNA degradation
+    assign mixture2 = mixture1 + C;
+
+    // move to diamond chamber so DNA can bind to magnetic particles
+    #MAP 'Diamond Chamber' '~'
```

Comment:
This needs to be a new component called `MAGNETIC SEPARATION CHAMBER`. And the lfr for it will be done by declaring a `storage` element, and using the `SIEVE (-)` operator.